app/model/jobs/ReindexSteps.scala

```diff
@@ -1,44 +1,64 @@
 package model.jobs
 
 import model.{Section, Tag}
-import play.api.libs.functional.syntax._
-import play.api.libs.json._
+import org.cvogt.play.json.Jsonx
 import repositories._
+import play.api.Logger
+import play.api.libs.json._
 import services.{Config, KinesisStreams}
+import scala.util.control.NonFatal
 
-// Tags
-
-case class ReindexTags(capiJobId: String) extends Step {
+case class ReindexTags() extends Step {
   override def process: Option[Step] = {
-    TagLookupCache.allTags.get.grouped(Config().reindexTagsBatchSize).foreach { tags =>
-      KinesisStreams.reindexTagsStream.publishUpdate(capiJobId, Tag.createReindexBatch(tags))
+    val total = TagLookupCache.allTags.get.size
```

Comment:
I was thinking about this. If we don't trust our caches we might as well just scan out the entire dynamo table for every request where we use the cache...

Would make tagmanager pretty much stateless (?) which is nice.